Solutions/HYAS Protect/Data/Solution_HyasProtectLogs.json

```diff
@@ -0,0 +1,17 @@
+{
+    "Name": "HYAS Protect",
+    "Author": "Hyas",
+    "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/blob/master/Logos/Hyas.svg\" width=\"75px\" height=\"75px\">",

```

Comment:
Hi @RamboV, please update this  link https://[raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/Hyas.svg](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/Hyas.svg)